x/concentrated-liquidity/pool_test.go

```diff
@@ -849,16 +846,14 @@ func (s *KeeperTestSuite) TestMigrateAccumulatorToScalingFactor() {
 	s.Require().NoError(err)
 
 	// Get claimable amount for position one before the migration
+	// NOTE: since the we enforce the scaling factor and never use the original scaling factor, this check is no longer applicable
```

Comment:
Which check?

Reply:
left over from previous iteration, removed in 63ea98f2121e7fd321c348acc0bc65ccb5a69df3